internal/cli/universal_login_customize.go

```diff
@@ -362,23 +368,30 @@ func (h *webSocketHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	}
 
 	if err = connection.WriteJSON(loadBrandingMsg); err != nil {
-		h.display.Errorf("failed to send branding data message: %v", err)
+		h.display.Errorf("Failed to send branding data message: %v", err)
+		h.display.Warnf("Try restarting the command.")
 		h.shutdown()
 		return
 	}
 
 	for {
 		var message webSocketMessage
 		if err := connection.ReadJSON(&message); err != nil {
-			h.display.Errorf("failed to read WebSocket message: %v", err)
+			if websocket.IsCloseError(err, websocket.CloseNormalClosure, websocket.CloseGoingAway) ||
+				websocket.IsUnexpectedCloseError(err, websocket.CloseAbnormalClosure) {
+				// The connection was closed.
+				break
+			}
```

Comment:
This logic is needed in order to prevent a panic where we keep trying to read from a closed connection due to the for loop.